frontend/src/App.js

```diff
@@ -1,111 +0,0 @@
-import React, { Component } from 'react';
-import axios from 'axios';
-import './App.css';
-import CreatureComponent from './components/CreatureComponent'
-
-// BASE URL
-const BASE_URL = "http://localhost:3005"
-
-class App extends Component {
-
-  constructor() {
-
-    super();
-
-    this.state = {
-
-      creatures: [],
-      newcreattext: '',
-      newcreatimg: '',
-      newcreatscary: ''
-
-    }
-
-  }
-
-  componentDidMount = () => {
-
-    axios.get(BASE_URL + "/api/creatures").then(res => {
-      this.setState({ creatures: res.data })
-    })
-
-  }
-
-  deletecreature = creatureID => {
-    // axios (DELETE)
-    axios.delete(BASE_URL + "/api/creatures/" + creatureID).then(res => {
-
-      this.setState({ creatures: res.data })
-
-    })
-  }
-
-  addcreature = creature => {
-    // axios (POST)
-    axios.post(BASE_URL + "/api/creatures", { newcreature: this.state.newcreattext, newimg: this.state.newcreatimg, scary: { level: this.state.newcreatscary } }).then(resp => {
-
-      this.setState({ creatures: resp.data })
-
-    })
-  }
-
-  updatecreature = (id, details) => {
-
-    axios.put(BASE_URL + "/api/creatures/" + id, details).then(res => {
-
-      this.setState({ creatures: res.data })
-
-    })
-
-  }
-
-  creaturename = event => {
-    this.setState({ newcreattext: event.target.value })
-  }
-
-  creatureimg = event => {
-    this.setState({ newcreatimg: event.target.value })
-  }
-
-  creaturescary = event => {
-    this.setState({ newcreatscary: event.target.value })
-  }
```

Comment:
these three funcs aren't bad, but they could be combined into one. I'm not saying that's necessarily better - it's a little bit less code, but also a little bit trickier to understand that way so I'd say it's probably sixes.